unstoppable-ios-app/domains-manager-ios/SupportingFiles/Localization/en.lproj/Localizable.strings

```diff
@@ -1111,6 +1111,10 @@ More tabs are coming in the next updates.";
 "IMPORT_MPC_WALLET_IN_PROGRESS_TITLE" = "Importing your\nMPC Wallet...";
 "IMPORT_MPC_WALLET_FINISHED_TITLE" = "All set, start using your MPC Wallet";
 "IMPORT_MPC_WALLET_FAILED_TITLE" = "Importing your\nMPC Wallet";
+"MPC_WALLET_SHARE_MULTI_CHAIN_DESCRIPTION" = "MPC Wallet has addresses across multiple blockchains.";
```

Comment:
All other sentences don't have `.` at the end. Is it needed here?

Reply:
It is required in this sentence